internal/monitors/kubernetes/cluster/metrics/pods.go

```diff
@@ -32,28 +31,32 @@ func datapointsForPod(pod *v1.Pod) []*datapoint.Datapoint {
 			time.Now()),
 	}
 
+	dimPropListForContainers := make([]*atypes.DimProperties, 0)
+	containersInPodByName := make(map[string]map[string]string)
+
 	for _, cs := range pod.Status.ContainerStatuses {
-		contDims := utils.CloneStringMap(dimensions)
-		contDims["container_id"] = strings.Replace(cs.ContainerID, "docker://", "", 1)
-		contDims["container_spec_name"] = cs.Name
-		contDims["container_image"] = cs.Image
-
-		dps = append(dps, datapoint.New(
-			"kubernetes.container_restart_count",
-			contDims,
-			datapoint.NewIntValue(int64(cs.RestartCount)),
-			datapoint.Gauge,
-			time.Now()))
+		contDims := getAllContainerDimensions(cs.ContainerID, cs.Name, cs.Image, dimensions)
```

Comment:
`cs.ContainerID` does not get the `docker://` prefix stripped off here or in the `getAllContainerDimensions` function.  Is that an oversight or intentional?

Reply:
I might have lost it during a rebase I did.  Added a test for this.